server/sandbox_run.go

```diff
@@ -265,10 +265,10 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 	// Don't use SELinux separation with Host Pid or IPC Namespace or privileged.
 	namespaceOptions := securityContext.GetNamespaceOptions()
 	if namespaceOptions == nil {
-		return nil, fmt.Errorf("no namespace options found")
+		namespaceOptions = &pb.NamespaceOption{}
 	}
 
-	if securityContext.GetNamespaceOptions().HostPid || securityContext.GetNamespaceOptions().HostIpc {
```

Comment:
change this to `if securityContext.GetNamespaceOptions().GetHostPid() ||  securityContext.GetNamespaceOptions().GetHostIpc()` same as my first comment